src/Microsoft.Health.Fhir.Shared.Api.UnitTests/Features/Resources/Bundle/BundleHandlerTests.cs

```diff
@@ -223,7 +261,7 @@ public async Task GivenATransactionBundleWithConditionalReferences_WhenResolved_
 
             foreach (var entry in bundle.Entry)
             {
-                IEnumerable<ResourceReference> references = entry.Resource.GetAllChildren<ResourceReference>();
+                List<ResourceReference> references = entry.Resource.GetAllChildren<ResourceReference>().ToList();
```

Comment:
Any specific reason to convert to List since we just intent to iterate over it? Just curious for my knowledge.

Reply:
To avoid possible multiple enumerations - https://www.jetbrains.com/help/resharper/PossibleMultipleEnumeration.html